newIDE/app/src/Utils/SimplifiedProjectJson.spec.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+import { getSimplifiedProjectJson } from './SimplifiedProjectJson';
+import { makeTestProject } from '../fixtures/TestProject';
+
+const gd: libGDevelop = global.gd;
+
+describe('SimplifiedProjectJson', () => {
+  it('should create a simplified project JSON with global objects and scenes', () => {
+    const { project } = makeTestProject(gd);
+    const simplifiedJson = getSimplifiedProjectJson(project);
+
+    expect(simplifiedJson).toMatchInlineSnapshot(`
+        Object {
+          "globalObjects": Array [
+            Object {
+              "behaviors": Array [],
+              "objectName": "GlobalTiledSpriteObject",
+              "objectType": "TiledSpriteObject::TiledSprite",
+            },
+            Object {
+              "behaviors": Array [],
+              "objectName": "GlobalTextObject",
+              "objectType": "TextObject::Text",
+            },
+          ],
+          "scenes": Array [
+            Object {
+              "objects": Array [
+                Object {
+                  "behaviors": Array [],
```

Comment:
how come we see the type in the json? Array / Object ? is that just the snapshot?

Reply:
Yes, Jest snapshot format